### PR TITLE
LPS-133412 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133412

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 1]
```diff
@@ -1159,4 +1159,50 @@ definition {
...
+				configurationScope = "Site Scope");
+		}
+
+		task ("Type the parameter in Staging Public Pages field") {
+			Type(
+				key_text = "Staging Public Pages",
+				locator1 = "TextInput#ANY",
+				value1 = "test.test");
+
+			SystemSettings.saveConfiguration();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1192]: Site.addVirtualHostsURLCP

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 2]
```diff
@@ -1159,4 +1159,50 @@ definition {
...
+		}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1194]: Add step to access to the sitemap as Test Test

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 3]
```diff
@@ -1159,4 +1159,50 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-109854. View the sitemap is not available to guest in staging site via incognito tab."
+	@priority = "4"
+	test ViewSiteMapNotAvailableToGuestInStagingSite {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1164]: CannotViewSiteMapOfStagingSiteAsGuest

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update case according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update case according to comments? Thanks

Hi @Tim-Cao, PR updated.
cc @liulukiwi

File: portal-web/test/functional/com/liferay/portalweb/functions/Open.function [hunk 4]
```diff
@@ -9,7 +9,11 @@ definition {
 
 		selenium.selectFrame("relative=top");
 
-		selenium.open();
+		var javascript = '''
+window.open("${locator1}","_self");
+		''';
+
+		selenium.runScript("${javascript}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/functions/Open.function:16]: Unneeded

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 5]
```diff
@@ -35,6 +35,46 @@ definition {
...
+				groupName = "Test Site Name (Staging)",
+				layoutName = "Public Page Name",
+				site = "false",
+				type = "content");
+		}
+
+		task ("Fill out the Staging Public Pages field in site configuration settings") {
+			Site.openSiteSettingsAdmin(siteURLKey = "test-site-name-staging");
+
+			Site.addVirtualHostsURLCP(stagingPublicPages = "test.test");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:54]: Use www.able.com or www.baker.com existing domain on Jenkins

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/6954d9e9fb1be98d284d9da8936644ee8d7ed2da...33557392f69b7c8a937ea5c2835dc841cd6fa1dc

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**